beam/www/beam/plugins/component.ts

```diff
@@ -0,0 +1,48 @@
+import type { ComponentResolver, Options } from 'unplugin-vue-components'
+
+import { HookConfig } from '@/types/config.js'
+import { getAppConfigs, mergeConfigs } from './hooks.js'
+
+export function getComponentPluginOptions(): Options {
+	// let globs: Options['globs']
+	// const components = getComponents()
+	// if ('*' in components) {
+	// 	globs = components['*']
+	// 	delete components['*']
+	// }
```

Comment:
@agritheory I was trying to allow component discovery using wildcards, but I'm not sure how the plugin expects me to pass the directory values. I'll do some more research later.